src/Parser/Warrior/Fury/Modules/BuffDebuff/EnrageUptime.js

```diff
@@ -8,27 +8,42 @@ import SpellLink from 'common/SpellLink';
 import { formatPercentage } from 'common/format';
 import StatisticBox from 'Main/StatisticBox';
 import Combatants from 'Parser/Core/Modules/Combatants';
+import Wrapper from 'common/Wrapper';
 
-class enrageUptime extends Analyzer {
+class EnrageUptime extends Analyzer {
   static dependencies = {
     combatants: Combatants,
   };
 
-  suggestions(when) {
-    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+  get enrageUptime() {
+    return this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+  }
 
-    when(enrageUptime).isLessThan(0.70)
+  get suggestionThresholds() {
+    return {
+      actual: this.enrageUptime,
+      isLessThan: {
+        minor: 0.07,
+        average: 0.065,
+        major: 0.06,
+      },
+      style: 'percentage',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds.actual).isLessThan(this.suggestionThresholds.isLessThan.minor)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your <SpellLink id={SPELLS.ENRAGE.id} /> uptime can be improved.</span>)
+        return suggest(<Wrapper>Your <SpellLink id={SPELLS.ENRAGE.id} /> uptime can be improved.</Wrapper>)
           .icon(SPELLS.ENRAGE.icon)
           .actual(`${formatPercentage(actual)}% Enrage uptime`)
           .recommended(`>${formatPercentage(recommended)}% is recommended`)
-          .regular(recommended - 0.05).major(recommended - 0.10);
+          .regular(this.suggestionThresholds.isLessThan.average).major(this.suggestionThresholds.isLessThan.major);
       });
   }
 
   statistic() {
-    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+    const enrageUptime = this.enrageUptime;
     return (
```

Comment:
no need to define it when you can just call it further down with ``this.enrageUptime``